sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -551,6 +551,14 @@ val GPU_COREDUMP_PIPE_PATTERN = conf("spark.rapids.gpu.coreDump.pipePattern")
       .integerConf
       .createWithDefault(2)
 
+  val ENABLE_VOLUNTARY_GPU_RELEASE_CHECK = conf("spark.rapids.gpu.voluntaryReleaseCheck")
+    .doc("If true, the plugin will check if voluntary release of GPU is forbidden, " +
+      "e.g. when GpuAggregateExec still have more output batches to offer." +
```

Comment:
missing a space at the end

Reply:
This piece of code is irrelavant to repartition based agg, so it will be removed by a subsequent commit